### PR TITLE
Add travis.yml

### PR DESCRIPTION
**Pull request by kevinburke**

### CODE REVIEW

**Comment by kevinburke** [comment]
You'll need to enable this on the Shyp account, I don't have permissions to do this anymore.

File: lib/lib.go [hunk 1]
```diff
@@ -32,7 +32,7 @@ func (v *Version) String() string {
 	} else if v.Major >= 0 {
 		return fmt.Sprintf("%d", v.Major)
 	} else {
-		return fmt.Sprintf("%!s(INVALID_VERSION)", v.Major)
+		return "%!s(INVALID_VERSION)"
```
Comment (dav) [inline lib/lib.go:35]: is this some kind of magic? what does `!s()` do?

**Comment by dav** [comment]
@kevinburke what need to be changed on what account?

File: lib/lib.go (hunk 1 above)
Comment (kevinburkeshyp) [reply lib/lib.go:35]: It's just what you get back from `fmt.Sprintf` and friends when there's an invalid format string.

**Comment by kevinburkeshyp** [comment]
Enable Travis CI builds. I can probably do it now.

**Pull request closed without merging**